content/docs/tasks/policy-enforcement/rate-limiting/index.md

```diff
@@ -58,7 +58,7 @@ so the configuration to enable rate limiting on both adapters is the same.
     Run the following command to enable rate limits.
 
     {{< text bash >}}
-    $ kubectl apply -f @samples/bookinfo/policy/mixer-rule-productpage-ratelimit.yaml@
+    $ kubectl apply -f examples/bookinfo/policy/mixer-rule-productpage-ratelimit.yaml@
```

Comment:
This change is not right. The original sequence is valid. Unfortunately, the will temporarily fail linting due to the fact the file only exists in the release-1.0 branch and has not been merged to master.

@mandarjog Do you need to merge this file to master, and will that be taken care of automatically?

Reply:
This file is in master

Reply:
I don't see it. I see it in release-1.0 but not master.

Reply:
Ok, I can revert this, I added it because of the linter.